source/api/search/2.0/index.md

```diff
@@ -367,12 +390,13 @@ And the user parameter was ignored when processing the request, the response wou
 ```
 
 
-#### 3.4.2. Search Term Snippets
+#### 3.4.2. Search Term Context
 {: #search-term-snippets}
 
+
 The simplest addition to the hit object is to add text that appears before and after the matching text in the annotation.  This allows the client to construct a snippet where the matching text is provided in the context of surrounding content, rather than simply by itself.  This is most useful when the service has word-level boundaries of the text on the canvas, such as are available when Optical Character Recognition (OCR) has been used to generate the text positions.
 
-The service _MAY_ add a `before` property to the hit with some amount of text that appears before the content of the annotation (given in `chars`), and _MAY_ also add an `after` property with some amount of text that appears after the content of the annotation.
+This is done by adding an entry to the `annotations` list with a `motivation` of "contextualizing", and a TextQuoteSelector with `prefix` and `suffix` of the text immediately before and after the matching content in the annotation. The selector has the URI of the annotation it refers to in the `source` property, to be matched against the `id` property of the annotations in `items`.
```

Comment:
I think we should reference the `exact` property here since we reference the `prefix` and `suffix` properties by name